src/components/Login/Login.jsx

```diff
@@ -1,22 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 
 import propTypes from 'prop-types';
 
-import { Link, useNavigate } from 'react-router-dom';
-
 import Header from '../Header/Header';
-
 import Input from '../../common/Input/Input';
-
 import Button from '../../common/Button/Button';
 
 import styles from './Login.module.css';
+import { addUserAction } from '../../store/users/actions';
+import { loginUser } from '../../services';
+
+const Login = ({ isValid, setIsValid }) => {
+	const [userData, setUserData] = useState({ email: '', password: '' });
```

Comment:
Here is much better to have separate states for the email and password
it is a good practice for the performance. Currently, your inputs will be updated simultaneously when the user fills out one of them. 

```suggestion
	const [email, setEmail] = useState("");
	const [password, setPassword] = useState("");
```